README.md

```diff
@@ -1,157 +1,67 @@
----
-page_type: sample
-products:
-- office-sp
-languages:
-- javascript
-- typescript
-extensions:
-  contentType: samples
-  technologies:
-  - SharePoint Framework
-  platforms:
-  - React
-  createdDate: 6/1/2020 12:00:00 AM
----
-
-copy of: https://github.com/pnp/sp-dev-fx-webparts/blob/master/samples/react-my-groups
-
-# My Groups
-## Update
-- Option to show all groups based on the selected letter or # (numbers)
-- You can choose how many groups to display in the property pane.
-- Ability to sort cards alphabetically or date the group was created.
-- French language icluded.
+# All communities
 
 ## Summary
 
-Using Microsoft Graph, this webpart generates all of the Office 365 groups that have been created in the SharePoint site.
-
+- This webpart generates all of the Office 365 groups that have been created in the SharePoint site
```

Comment:
Please change to The webpart show all SharePoint teams site.

Reply:
Sure